src/dashboards/components/DashboardPage.tsx

```diff
@@ -72,31 +88,41 @@ class DashboardPage extends Component<Props> {
           <Page titleTag={this.pageTitle} testID="dashboard-page">
             <LimitChecker>
               <HoverTimeProvider>
-                <DashboardHeader onManualRefresh={onManualRefresh} />
-                <RateLimitAlert alertOnly={true} location="dashboard page" />
-                <VariablesControlBar />
-                <ErrorBoundary>
-                  <DashboardComponent manualRefresh={manualRefresh} />
-                </ErrorBoundary>
+                <Switch>
+                  <Route
+                    path={dashRoute}
+                    render={() => (
+                      <SingleDashboardPage
+                        manualRefresh={manualRefresh}
+                        onManualRefresh={onManualRefresh}
+                      />
+                    )}
+                    exact
```

Comment:
this needs to be `exact={isFlagEnabled()}` right?

Reply:
does it?

Reply:
figured it out. seems like routes are impervious to redux changes anyways